### DISCUSSION TITLE
Duvida: é possivel utilizar o github pages em repositorio privado?

### DISCUSSION THREAD

**Discussion by josevictormoreno** [frontendbr/forum#2292] [Perguntas e Respostas]
Da pra utilizar o github pages em um repositorio privado, ou deve ser publico? se for possível, a pagina ficaria privada tbm?

**Comment by felquis** [comment]
Eai bele? É possível sim, mas é bom vc ler essa página que contem todos os detalhes https://docs.github.com/pt/pages/getting-started-with-github-pages/about-github-pages

**Reply by nbkhope** [reply]
> private repositories with GitHub Pro, GitHub Team, GitHub Enterprise Cloud, and GitHub Enterprise Server

> Organizations that use GitHub Enterprise Cloud can also publish sites privately by managing access control for the site.

O repositorio sendo publico ou privado é uma coisa. O site sendo público ou privado é outra.

Como diz a documentação, para usar repositórios privados, tem que ter um dos seguintes: "GitHub Pro, GitHub Team, GitHub Enterprise Cloud, and GitHub Enterprise Server". Note que mesmo que o repositorio seja privado, o site pode ser publico: "GitHub Pages sites are publicly available on the internet, even if the repository for the site is private."

Como diz a documentação, sites privados podem ser publicados com o "GitHub Enterprise Cloud".
